### ISSUE TITLE
Proof-Read the readme

### ISSUE THREAD

**Issue by turbopape** [fekr/postagga#16]

**Comment by mmafrar** [comment]
@turbopape Interested on contributing to this issue

**Comment by turbopape** [comment]
Yes please Go ahead, but you might want to submit your pull request in October, to be able to enter [Hacktoberfest](https://hacktoberfest.digitalocean.com/) !

**Comment by naxical** [comment]
I'll proof read it.

**Comment by mattrlee3** [comment]
Ill also proofread it and send a PR once in October

**Comment by Eddenm** [comment]
Is this issue still open? Can I review it for you?
